### ISSUE TITLE
Conv3D updates?

### ISSUE THREAD

**Issue by MasalaKimchi** [ZPZhou-lab/tfkan#4]
Thank you for the tensorflow implementation KAN. I was wondering if you are planning to have Conv3DKAN, an extension of Conv2DKAN in near future.

best

**Comment by ZPZhou-lab** [comment]
Thanks for feedback. I update the code and implement more convolution operations including `Conv1DKAN`, `Conv2DKAN`, `Conv3DkAN`. You can use them from `tfkan.layers`:

```python
import tensorflow as tf
from tfkan.layers import Conv3DKAN

# build a Conv3D layer
input_shape = (16, 32, 28, 28, 3)
x = tf.random.normal(shape=input_shape)
layer = Conv3DKAN(filters=8, kernel_size=3, strides=(1, 2, 2), padding="same")
y = layer(x)
print(y.shape) # (16, 32, 14, 14, 8)
```

To make sure the `channels` is the last axis of your input tensors and the 3-dims after `batch_size` can be `(spatial_1, spatial_2, spatial_3)` (usually `(num_steps, height, width)` of a video)

Hope this can help you. Best

**Comment by MasalaKimchi** [comment]
This is awesome, thank you so much! I will look forward to developing Conv3DKAN based neural network!
